Dfe.Academies.External.Web.UnitTest/Services/ConversionApplicationCreationServiceTests.cs

```diff
@@ -111,4 +114,35 @@ public async Task AcademisationCreationService___UpdateDraftApplication___ChairR
         // assert
         Assert.DoesNotThrowAsync(() => recordModelService.UpdateDraftApplication(trustApplicationDto));
     }
+
+    [Test]
+    public async Task AcademisationCreationService___AddSchoolToApplication___Success()
```

Comment:
Can we use the test name format:

`MethodName___PreCondition1_PreCondition2___ExpectedOutcome()`

Which in this case would be:

`AddSchoolToApplication___ApiReturns200___Success()`

We'll need to add some handling for other API responses as well.